libraries/default.rb

```diff
@@ -670,13 +670,15 @@ def install_otherpubs_service_series_impl_config(current_deploy_root)
       end
     end
 
+    # nodes that don't dispatch should also not do hearbeat service health checks
```

Comment:
Change the method name? e.g. configure_service_registry?

Reply:
@rute-santos  I'm not sure what you mean by change the name.

Reply:
Sorry, the comment is related to the line below :) The method is 'set_service_registry_dispatch_interval', but now we are configuring more than the dispatch interval.